lib/ansible/modules/network/nxos/nxos_interface.py

```diff
@@ -279,7 +279,7 @@ def get_interface(intf, module):
 
     command = 'show interface {0}'.format(intf)
     try:
-        body = execute_show_command(command, module)[0]
+        body = run_commands(module, [command])[0]
```

Comment:
@mikewiebe looks like a good plan. it seems pipelining `| json` with `show run` command returns empty string which makes the module broken. but removing json output would break the other cases.